PennMobile/LaundryPrototype/Views/LaundryRowView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  LaundryRowView.swift
+//  PennMobile
+//
+//  Created by Nathan Aronson on 10/20/25.
+//  Copyright © 2025 PennLabs. All rights reserved.
+//
+
+
+import SwiftUI
+
+struct LaundryRowView: View {
+    
+    let hall: LaundryHallId
+    let isSelected: Bool
+    let canSelect: Bool
+    let toggle: () -> Void
+    
+    var body: some View {
+        HStack {
+            Text(hall.name)
+                .font(.body)
+                .foregroundColor(.primary)
```

Comment:
`.foregroundStyle` aaaaaaaaaaaaa
```suggestion
                .foregroundStyle(.primary)
```